CHANGELOG.md

```diff
@@ -1,5 +1,111 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.14.6
+<!-- generated comparing v5.14.5..master -->
+_Aug 23, 2023_
+
+A big thanks to the 21 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Added [Accordion](https://mui.com/joy-ui/react-accordion/) component to Joy UI (#38164) @siriwatknp
+- 🎉 Added Material You Input and ButtonBase components to `material-next` (#38319) @DiegoAndai @mj12albert
+- 🔋 Added zero-runtime support for [Linaria](https://linaria.dev/) (#38378) @brijeshb42
```

Comment:
@brijeshb42 is my understanding is correct?

Reply:
It doesn't feel like we added support for it yet. IMHO, this sets wrong expectation for developers, for example, until there is documentation, we can't say a feature is available IMHO.

Maybe?

```suggestion
- 🔋 First iteration on a CSS-in-JS compatible with Server Components (#38378) @brijeshb42
```

Reply:
☝️ Maybe missing what "a CSS-in-JS" is? Here's an option:

> 🔋 First iteration on a CSS-in-JS _**engine**_ compatible with Server Components